### ISSUE TITLE
Failed to find 'glualint' path, even tho it is there.

### ISSUE THREAD

**Issue by deltamolfar** [Goz3rr/vscode-glualint#21]
Hi, I hope I didn't get a skill issue, but I downloaded glualint, and the vscode-glualint, added former to the path, and it did work for the few days. Then it decided not to :D

I'm using Debian 12, and regular vscode.

*vs code error*
![2024-06-08_01-54](https://github.com/Goz3rr/vscode-glualint/assets/72973198/21c8d1ae-a307-4778-a642-35bd04a8366d)

*glualint itself work as expected*
![2024-06-08_01-57](https://github.com/Goz3rr/vscode-glualint/assets/72973198/0c2b58b3-1c5a-4842-9da1-3a631795b2d4)

*my .bashrc*
![2024-06-08_02-02](https://github.com/Goz3rr/vscode-glualint/assets/72973198/18a0f70d-4af5-465b-b8ce-0732e1bd6277)

**Comment by Motoess** [comment]
I have the exact same issue. Lets pray this gets fixed soon.

**Comment by Goz3rr** [comment]
I have seen general weirdness around vscode not properly respecting changes to the path, but that was on Windows. You can confirm it's working as it should be opening a terminal inside vscode itself (View > Terminal) and running the `glualint` command in there. If that works then there's something else really weird going on.

A possible workaround is to set the `glualint.linter` setting to the absolute path to your executable. In your case it would probably be `/home/deltamolfar/Downloads/glualint/glualint`

**Comment by deltamolfar** [comment]
Just checked. Turned out it's working again as expected, even tho I've never changed anything. (Also I recall that I've used glualint from vscode's terminal, and it worked fine).
(Just disabled it since this report, and now enabled it again to check)

@Motoess maybe also try to disable and enable it back again? (maybe after the restart?)

**Comment by deltamolfar** [comment]
Yeah it happened again. Failed to find - and glualint still works in internal terminal. Just set up global path - works fine
